manifests/weave/davenet/bns/genesis.json

```diff
@@ -128,13 +200,195 @@
       }
     },
     "msgfee": [
-      {"fee": "100 IOV", "msg_path": "username/register_token"},
-      {"fee": "10 IOV", "msg_path": "username/change_token_targets"},
-      {"fee": "50 IOV", "msg_path": "username/transfer_token"}
+      {
+        "fee": "100 IOV",
+        "msg_path": "username/register_token"
+      },
+      {
+        "fee": "10 IOV",
+        "msg_path": "username/change_token_targets"
+      },
+      {
+        "fee": "50 IOV",
+        "msg_path": "username/transfer_token"
+      }
     ],
-    "update_validators": {
-      "addresses": [
-        "cond:multisig/usage/0000000000000001"
+    "governance": {
+      "electorate": [
+        {
+          "title": "Test manually managed by Cloe",
+          "admin": "BA1DBBA7C4A04F39A98FA5B4DBA8BC6981D7D40E",
+          "electors": [
+            {
+              "_name": "Anton",
```

Comment:
Sometimes you are using `//name` and sometimes `_name`. I think we should stick to a single pattern.